tests/test_arrow.py

```diff
@@ -2266,6 +2266,25 @@ def test_untranslated_granularity(self, mocker):
         with pytest.raises(ValueError):
             arw.humanize(later, granularity="week")
 
+    # Bulgarian is an example of a language that overrides _format_timeframe
+    # Applicabale to all locales. Note: Contributors need to make sure
+    # that if they override describe or describe_mutli, that delta
+    # is truncated on call
```

Comment:
I wonder if there is a way to detect if a function is override in a subclass and then check if the value is an `int` (meaning it was truncated) rather than a `float` (meaning it was not truncated). What do you think?